tests/types/test_numpy.py

```diff
@@ -0,0 +1,131 @@
+import pytest
+from psycopg.adapt import PyFormat
+from psycopg.types.numpy import register_numpy
+
+pytest.importorskip("numpy")
+
+import numpy as np  # noqa: E402
+
+# np Numeric types
+#
+# np.int8 (-128, 127) -> smallint (alias int2)
+# np.int16 (-32768, 32767) -> smallint
+# np.int32 (-2147483648, 2147483647) -> integer (alias int4)
+# np.int64 (-9223372036854775808, 9223372036854775807) -> bigint (alias int8)
+# np.uint8 (0 - 255)
```

Comment:
This table is not complete: either we fill it in altogether or we drop it (in case it was just a note for you as you were looking into the types).